### PR TITLE
qeels docstrings

### PR DESCRIPTION
**Pull request by veengren-s**

### CODE REVIEW

**Comment by veengren-s** [comment]
WIP

File: qeels/qeels/engine/results.py [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 
 def save_results(file_path, headers, data):
+    """
+    Saves the data to the provided file path
+    """
```
Comment (luctowers) [inline qeels/qeels/engine/results.py:7]: For these short docstrings you can just make em a single line

File: qeels/qeels/gui/window_main.py [hunk 2]
```diff
@@ -359,6 +375,9 @@ def save_results(self):
             save_results(save_path, headers, data)
 
     def detect(self):
+        """
+        Performs peak detection on spectrogram
+        """
```
Comment (luctowers) [inline qeels/qeels/gui/window_main.py:380]: again shorter docstrings can go on a single line

**Review by luctowers — APPROVED** [review]
lgtm thanks!

**Pull request merged**